web/controllers/api/v1/habit_controller.ex

```diff
@@ -52,6 +52,22 @@ defmodule Habits.API.V1.HabitController do
   end
 
   @doc """
+  Update the name of a habit in the current account
+  """
+  def update(conn, %{"id" => habit_id, "name" => name}, current_account) do
+    habit =
+      current_account
+      |> assoc(:habits)
+      |> Repo.get(habit_id)
+      |> Habit.changeset(%{name: name})
+      |> Repo.update!
```

Comment:
I'm considering extracting a `Habit.update(habit, attributes)` function to encapsulate some of this, but I worry that I'm just trying to turn this into ActiveRecord...